2018-03-25 20:14:51,725 INFO: load_user: user1 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:14:51,816 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:14:51,816 INFO: load_user: user1 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:14:57,282 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:14:57,282 INFO: load_user: user1 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:14:57,349 INFO: load_user: user1 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:15:27,376 INFO: Flask Starter startup [in /Users/patryan/Development/mygithub/flask-starter/app/__init__.py:42]
2018-03-25 20:15:30,976 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:15:30,977 INFO: load_user: user1 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:15:31,087 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:15:31,088 INFO: load_user: user1 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:15:31,617 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:15:31,617 INFO: load_user: user1 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:15:31,692 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:15:31,694 INFO: load_user: user1 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:15:58,842 INFO: load_user: user1 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:15:58,934 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:16:01,368 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:16:07,228 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:16:07,363 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:16:07,451 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:16:07,452 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:17:10,624 INFO: Flask Starter startup [in /Users/patryan/Development/mygithub/flask-starter/app/__init__.py:42]
2018-03-25 20:17:16,394 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:17:16,395 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:17:16,488 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:17:16,488 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:28:21,432 INFO: Flask Starter startup [in /Users/patryan/Development/mygithub/flask-starter/app/__init__.py:42]
2018-03-25 20:28:46,899 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:28:46,899 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:28:47,000 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:28:47,001 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:28:47,835 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:28:47,948 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:28:47,949 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:28:49,740 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:28:49,806 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:28:49,887 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:28:49,888 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:29:46,744 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:29:46,835 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:32:14,929 INFO: Flask Starter startup [in /Users/patryan/Development/mygithub/flask-starter/app/__init__.py:42]
2018-03-25 20:32:16,700 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:32:16,752 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:32:46,844 INFO: Flask Starter startup [in /Users/patryan/Development/mygithub/flask-starter/app/__init__.py:42]
2018-03-25 20:32:48,845 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:33:12,285 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:33:12,410 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:33:12,496 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:33:12,497 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:37:28,772 INFO: Flask Starter startup [in /Users/patryan/Development/mygithub/flask-starter/app/__init__.py:42]
2018-03-25 20:37:30,323 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:37:30,415 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:37:30,416 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:40:47,861 INFO: Flask Starter startup [in /Users/patryan/Development/mygithub/flask-starter/app/__init__.py:42]
2018-03-25 20:40:49,092 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:40:49,166 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:40:49,166 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:41:04,599 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:41:04,651 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:41:04,652 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:41:17,075 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:41:17,102 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:41:24,124 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:41:24,252 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:41:49,386 INFO: Flask Starter startup [in /Users/patryan/Development/mygithub/flask-starter/app/__init__.py:42]
2018-03-25 20:41:50,290 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:41:50,368 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:41:50,368 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:41:50,440 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:41:50,440 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:41:52,467 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:41:52,523 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:41:52,523 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:41:52,609 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:41:58,983 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:41:59,118 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:41:59,192 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
